src/mockServiceWorker.js

```diff
@@ -206,13 +206,6 @@ async function getResponse(event, client, requestId) {
     return passthrough()
   }
 
-  // Bypass requests with the explicit bypass header.
-  // Such requests can be issued by "ctx.fetch()".
-  const mswIntention = request.headers.get('x-msw-intention')
```

Comment:
Both bypassed and passthrough requests _must_ emit the `REQUEST` event to the client so the client (worker context) stores that request in memory. Then, it can find the request instance based on the `requestId` when receiving the `RESPONSE` event, which right now both mocked and bypassed responses do send correctly.